configs/dcn/README.md

```diff
@@ -1,4 +1,8 @@
-# Deformable Convolutional Networks
+# DCN
```

Comment:
We should divide the configs in this directory into two directories, one for DCN, one for DCN v2 so that one README only contains configs of one paper.

Reply:
OK. I will check it right away.